src/components/header/Header.jsx

```diff
@@ -30,6 +37,7 @@ const Header = (props) => {
                 <div>
                   <Link to="/" className={className}>Home</Link>
                   <Link to="/create-article" className={className}>Create post</Link>
+                  <a href="#" onClick={logout} className={className}>Log out</a>
```

Comment:
@leksyib why use an anchor tag here?

Reply:
@9jaswag  <Link/> tag requires a `to` prop. So I decided to use the anchor tag. I could also use a <Link/> tag but before the component can know the route it's in, we'll have to pass `history` prop from every other component using the Header component.

Reply:
@leksyib what about just using a button?